pkg/capabilities/crypto/crypto.go

```diff
@@ -38,7 +37,7 @@ func (e CertificateEncoding) MarshalJSON() ([]byte, error) {
 //     (intermediates first, root last). If empty, certificate is assumed trusted.
 //   - not_after: string in RFC 3339 time format, to check expiration against.
 //     If None, certificate is assumed never expired.
-func VerifyCert(h *cap.Host, cert Certificate, certChain []Certificate, notAfter string) (bool, error) {
+func VerifyCert(h *cap.Host, cert Certificate, certChain []Certificate, notAfter string) (CertificateVerificationResponse, error) {
```

Comment:
I would change `CertificateVerificationResponse` to be pointer

Reply:
Indeed! done.